packages/webcomponents/src/components/payouts-list/payouts-list-core.tsx

```diff
@@ -0,0 +1,151 @@
+import { Component, Host, h, Prop, State, Watch, Event, EventEmitter } from '@stencil/core';
+import {
+  PagingInfo,
+  Payout,
+  PayoutStatuses,
+  PayoutStatusesSafeNames,
+  pagingDefaults
+} from '../../api';
+import { formatCurrency, formatDate, formatTime } from '../../utils/utils';
+import { tableExportedParts } from '../table/exported-parts';
+
+@Component({
+  tag: 'payouts-list-core',
+  styleUrl: 'payouts-list.scss',
+})
+
+export class PayoutsListCore {
+  @Prop() getPayouts: Function;
+  @State() payouts: Payout[] = [];
+  @State() loading: boolean = true;
+  @State() errorMessage: string;
+  @State() paging: PagingInfo = pagingDefaults;
+  @State() params: any
+  @Event({
+    eventName: 'payout-row-clicked',
+    bubbles: true,
+  }) rowClicked: EventEmitter<Payout>;
+
+  componentWillLoad() {
+    if (typeof this.getPayouts === 'function') {
```

Comment:
Similar to my note on the payments list PR - can we refactor this like you did in the other PR? 

```
componentWillLoad() {
    if (this.getPayouts) {
      this.fetchData();
    }
  }
```

and then remove the duplicate check for this logic on line 73?